api-inference-community/docker_images/common/tests/test_api_audio_to_audio.py

```diff
@@ -49,11 +51,15 @@ def test_simple(self):
             response.status_code,
             200,
         )
-        self.assertEqual(response.headers["content-type"], "audio/flac")
-        audio = ffmpeg_read(response.content)
+        self.assertEqual(response.headers["content-type"], "application/json")
+        audio = json.loads(response.content)
 
-        self.assertEqual(len(audio.shape), 1)
-        self.assertGreater(audio.shape[0], 1000)
+        self.assertTrue(isinstance(audio, list))
+        self.assertEqual(set(audio[0].keys()), {"blob", "content-type", "label"})
+
+        data = base64.b64decode(audio[0]["blob"])
+        wavform = ffmpeg_read(data)
+        self.assertGreater(wavform.shape[0], 1000)
 
```

Comment:
Nit: should we add tests for sampling rate and the annotations?

Reply:
We can test the type for sure !

Reply:
That would be good idea :)

Reply:
Done !